### ISSUE TITLE
Develop a library of example codemods / starting points:

### ISSUE THREAD

**Issue by jamestalmage** [jamestalmage/generator-codemod#5]
Give users a list of examples to choose from, like:
- rename a variable
- reorder arguments to a method
- ...

Make it easy to add new templates to the list of examples to encourage community involvement in developing good examples.

When accepting a new template / adding something to the list: 
1. It shouldn't do the same thing as another example. 
2. It shouldn't be something users could easily modify an existing example to do.

**Comment by sindresorhus** [comment]
Do you intend to add a prompt in the generator to choose a starting point / example? Or is this purely a documentation thing?

**Comment by jamestalmage** [comment]
Yeah - the prompt thing. I'd want a good default one that we decided was the "best" for people who already knew what they were doing, so you can bypass that prompt quickly.

If we think prompt is unhelpful / too costly, then  "a documentation thing" doesn't make sense. We should just contribute examples back to jscodeshift at that point.

**Comment by jfmengels** [comment]
Sounds like a good idea, if we can provide some helpful starters. I'd prepare some astexplorer links in the comments of the generated file too, so that the user can play with it on the website.
